packages/core/utils/src/product/csv-normalizer.ts

```diff
@@ -0,0 +1,520 @@
+import {
```

Comment:
thought: lots of good stuff in here, that could be used for a more generic parser

Reply:
Makes sense 👍